docs/development/firmware/building.mdx

```diff
@@ -72,7 +72,7 @@ The build system is modular. Adding a new board variant for an already supported
 
 Gitpod offers an alternative method for compiling the firmware without installing Visual Studio Code. It is a web browser-based online IDE that can be used with a GitHub account. Gitpod provides a free tier suitable for light use, such as compiling custom firmware.
 
-1. Go to (https://gitpod.io#https://github.com/meshtastic/firmware)[https://gitpod.io#https://github.com/meshtastic/firmware].
+1. Go to <a href="https://gitpod.io#https://github.com/meshtastic/firmware">https://gitpod.io#https://github.com/meshtastic/firmware</a>.
```

Comment:
This one could use markdown too, there was just and extra space between the ) and the [

Reply:
Square brackets go before round parentheses(?). I think it works now. 🤞